server/src/logic/utils/model.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiError, ERRORS } from "./../../models/index";
+
+export class ModelUtils {
```

Comment:
What does the modelutils Class? If I get it right than Object.assign(..) does the same!?